flink-libraries/flink-table/pom.xml

```diff
@@ -92,6 +92,11 @@ under the License.
 			</exclusions>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.hive</groupId>
```

Comment:
I haven't had a look at the entire code yet, but the `flink-table` should not have a dependency to Hive. We should move the Hive dependent code to `flink-hcatalog`. Users that want to use any Hive functionality can then add this module to their dependencies. Similar to `KafkaTableSource`.